### PR TITLE
 Remove old & unused HTML files

### PR DESCRIPTION
**Pull request by wilzbach**
Check for yourself:  https://dlang.org/dcompiler.html

__edit__: found a couple of more old files

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -321,9 +321,7 @@ CHANGELOG_DDOC=${DDOC} changelog/changelog.ddoc $(NODATETIME)
 CHANGELOG_PRE_DDOC=${CHANGELOG_DDOC} changelog/prerelease.ddoc
 CHANGELOG_PENDING_DDOC=${CHANGELOG_DDOC} changelog/pending.ddoc
 
-PREMADE=appendices.html articles.html fetch-issue-cnt.php howtos.html \
```
Comment (wilzbach) [inline posix.mak]: There was no `articles.html` anymore.

**Comment by CyberShadow** [comment]
IIRC the intention of keeping these was for testing on setups without .htaccess support, such as right from the filesystem. Are these not linked from anywhere on dlang.org?

**Comment by wilzbach** [comment]
> Are these not linked from anywhere on dlang.org?

- articles.html: has been replaced by `articles.dd` (it's apparently not PREMADE anymore) -> https://dlang.org/articles.html
- they other files only appear in `PREMADE`

```
> ag "(appendices|dcompiler|debugger|howtos|language-reference).html"
posix.mak
324:PREMADE=appendices.html articles.html fetch-issue-cnt.php howtos.html \
325:	language-reference.html robots.txt .htaccess .dpl_rewrite_map.txt \
```

File: posix.mak (hunk 1 above)
Comment (JinShil) [reply posix.mak]: Some of the articles were moved to the wiki (by yours truly) so they could be more easily maintained, as many of them had rotted horribly.

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,3 @@
-!appendices.html
-!articles.html
-!dcompiler.html
-!howtos.html
-!debugger.html
-!language-reference.html
```
Comment (JinShil) [inline .gitignore]: Probably caused by me:
https://github.com/dlang/dlang.org/pull/496
https://github.com/dlang/dlang.org/pull/514
Sorry!

**Pull request merged**